static/keyboard.js

```diff
@@ -1,66 +1,101 @@
 (function() {
+    function focusSearchInput() {
+        // On the index page, we have a "#search" input. If we are on this page, we want to go back
+        // to this one and not the one in the header navbar.
+        var searchInput = document.getElementById("search");
+        if (searchInput) {
+            searchInput.focus();
+        } else {
+            document.getElementById("nav-search").focus()
+        }
+    }
+
+    function focusFirstSearchResult() {
+        var elem = document.querySelector(".recent-releases-container a.release");
+        if (elem) {
+            elem.focus();
+        }
+    }
+
+    function getWrappingLi(elem) {
+        while (elem.tagName !== "LI") {
+            elem = elem.parentElement;
+        }
+        return elem;
+    }
+
+    function focusOnLi(li) {
+        var elem = li.querySelector(".release");
+        if (elem) {
+            elem.focus();
+        }
+    }
+
     function getKey(ev) {
         if ("key" in ev && typeof ev.key != "undefined") {
             return ev.key;
         }
         return String.fromCharCode(ev.charCode || ev.keyCode);
     }
 
-    var active = null;
+    function checkIfHasParent(elem, className) {
+        while (elem && elem.tagName !== "BODY") {
+            elem = elem.parentElement;
+            if (elem.classList.constains(className)) {
```

Comment:
```suggestion
            if (elem.classList.contains(className)) {
```

The typo suggests this code path wasn't exercised during testing. Can you make sure to test a case when `checkIfHasParent` returns true?